src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -129,7 +140,12 @@ internal Binder Binder
             SyntaxToken identifierToken,
             CSharpSyntaxNode context)
         {
-            return new OutLocalSymbol(containingSymbol, scopeBinder, enclosingBinderOpt, typeSyntax, identifierToken, context);
+            if (typeSyntax.IsVar)
+            {
```

Comment:
This is consistent with behavior in MakeLocal method above and with implementation for regular locals. I am keeping the code. Thanks.